analytics_data_api/v0/tests/views/test_learners.py

```diff
@@ -25,10 +25,16 @@ def setUp(self):
         """Creates the index and defines a mapping."""
         super(LearnerAPITestMixin, self).setUp()
         self._es = Elasticsearch([settings.ELASTICSEARCH_LEARNERS_HOST])
-        # delete index if for some reason the index wasn't deleted in tearDown
-        if self._es.indices.exists(index=settings.ELASTICSEARCH_LEARNERS_INDEX):
-            self._es.indices.delete(index=settings.ELASTICSEARCH_LEARNERS_INDEX)
-        self._es.indices.create(index=settings.ELASTICSEARCH_LEARNERS_INDEX)
+
+        for index in [settings.ELASTICSEARCH_LEARNERS_INDEX, settings.ELASTICSEARCH_LEARNERS_UPDATE_INDEX]:
+            # delete index if for some reason the index wasn't deleted
+            def delete_index(to_delete):
+                if self._es.indices.exists(index=to_delete):
+                    self._es.indices.delete(index=to_delete)
+            delete_index(index)
```

Comment:
I was debating whether to include it or not.  I decided to include it in case the index wasn't deleted before our new snazzy `addCleanup` functionality.  I'm thinking now that we can assume that the system is clean and that there isn't a need for calling `delete_index()` explicitly; I'll remove it.